app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -28,12 +34,59 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
       });
   } else {
     miqService.sparkleOn();
-
-    $http.get('/network_router/network_router_form_fields/' + networkRouterFormId)
-      .then(getNetworkRouterFormData)
-      .catch(miqService.handleFailure);
+    API.get("/api/network_routers/" + networkRouterFormId + "?attributes=name,ems_id,admin_state_up,cloud_network_id,extra_attributes,cloud_tenant,ext_management_system,cloud_subnets").then(function(data) {
+      vm.networkRouterModel.name = data.name;
+      vm.networkRouterModel.cloud_network_id = data.cloud_network_id;
+      vm.networkRouterModel.ems_id = data.ext_management_system.id;
+      vm.networkRouterModel.ems_name = data.ext_management_system.name;
+      vm.networkRouterModel.cloud_tenant_name = data.cloud_tenant.name;
+      vm.networkRouterModel.extra_attributes = data.extra_attributes;
+    }).then(function() {
```

Comment:
this line should probably not be here?

`data` is only defined in the first function, not the second (hence the eslint warning)

Reply:
Right! Done.